File: InstructorsApp/Models/Instructor.cs
using System.ComponentModel.DataAnnotations;

namespace InstructorsApp.Models
{
	public class Instructor
	{
		[Key]
		public int Id { get; set; }

		[Required]
		public string FirstName { get; set; }

		[Required]
		public string MiddleName { get; set; }

		[Required]
		public string LastName { get; set; }
	}
}

File: InstructorsApp/ClientApp/src/app/instructor.ts
export class Instructor {
  id: number;
  firstName: string;
  middleName: string;
  lastName: string;

  constructor(firstName: string, middleName: string, lastName: string) {
    this.firstName = firstName;
    this.middleName = middleName;
    this.lastName = lastName;
  }
}
